src/foam/nanos/notification/email/ChainedPropertyService.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.notification.email',
+  name: 'ChainedPropertyService',
```

Comment:
This model seems incomplete...

Reply:
lol ... ok let me think about it.

Reply:
Should be pretty easy. The sendEmail method just iterates over the FObjectArray and call sendEmail on each of those.